### PR TITLE
Create Colors-and-Backgrounds.md

### PR DESCRIPTION
**Pull request by AmrutaJayanti**
## Description

Added Colors-and-Backgrounds.md

### Issues Related:

Closes: #317 

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [x] Other (specify): Adding Solution

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: docs/CSS/css-basics/Colors-and-Backgrounds.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+In CSS, controlling colors and background properties is fundamental for styling web pages. This includes setting text colors, background colors, images, gradients, and more. Below are the main properties and techniques for handling colors and backgrounds in CSS.
+
+**1. Text Color**
+
+The color property sets the color of the text. Colors can be defined using various formats, such as named colors, HEX codes, RGB, RGBA, HSL, and HSLA.
+
+Hex - Codes :
+
+![HexCode](https://tse4.mm.bing.net/th?id=OIP.9ipFzqjVN-PlJKwrpuAiuQHaFM&pid=Api&P=0&h=180)
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-basics/Colors-and-Backgrounds.md:11]: change size of `img`

File: docs/CSS/css-basics/Colors-and-Backgrounds.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+RGB (Red-Green-Blue):
+
+<img src="https://tse4.mm.bing.net/th?id=OIP.zBlyNAyf8fFyqp7_vdv2mgHaCx&pid=Api&P=0&h=180" alt="RGB" height="500" width="400"/>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-basics/Colors-and-Backgrounds.md:15]: remove height, width or maintain size of `img`

File: docs/CSS/css-basics/Colors-and-Backgrounds.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+HSL (Hue-Saturation-Lightness):
+
+<img src="https://giggster.com/guide/static/fed42130c194b0c240a4ec10408adf97/8282f/hsl-cover-2.png" alt="HSL" height="200" width="200"/>
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-basics/Colors-and-Backgrounds.md:19]: change size of `img` based on content

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
change code based on suggestions

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar I made the changes . Please approve

**Review by Ajay-Dhangar — APPROVED** [review]
done

**Pull request merged**